packages/perseus/src/widgets/__stories__/expression.stories.tsx

```diff
@@ -46,6 +46,7 @@ const WrappedKeypadContext = ({
                             apiOptions={{
                                 isMobile: isMobile,
                                 customKeypad: customKeypad,
+                                onFocusChange: () => {},
```

Comment:
Wonder if we could use a SB action here like we do in a few other stories.

```suggestion
                                onFocusChange: action("onFocusChange"),
```